client/src/app/pages/sbom-details/packages-by-sbom.tsx

```diff
@@ -167,7 +168,7 @@ export const PackagesBySbom: React.FC<PackagesProps> = ({ sbomId }) => {
                       <Flex>
                         <FlexItem
                           spacer={{ default: "spacerSm" }}
-                        >{`${item.decomposedPurl?.name}${item.decomposedPurl?.namespace ? "/" + item.decomposedPurl?.namespace : ""}`}</FlexItem>
+                        >{`${item.decomposedPurl?.name}${item.decomposedPurl?.namespace ? `/${item.decomposedPurl.namespace}` : ""}`}</FlexItem>
```

Comment:
```suggestion
                       >
                         {(() => {
                            const name = item.decomposedPurl?.name || "";
                            const namespace = item.decomposedPurl?.namespace
                              ? `/${item.decomposedPurl.namespace}`
                              : "";
                            return `${name}${namespace}`;
                          })()}
                        </FlexItem>
```
This is hard to read maybe for better readability ?